distr_tracing/distr_tracing_rn/distr-tracing-rn-2-9.adoc

```diff
@@ -0,0 +1,196 @@
+
+* Expose the Jaeger Query gRPC endpoint on the Query Frontend service.
+
+* Support multitenancy without Gateway authentication and authorization.
+
+//=== Technology Preview features
+//not for 2.9
+
+=== Bug fixes
+
+* Before this update, the {TempoOperator} was not compatible with disconnected environments. With this update, the {TempoOperator} supports disconnected environments. (link:https://issues.redhat.com/browse/TRACING-3145[TRACING-3145])
+
+* Before this update, the {TempoOperator} with TLS failed to start on {product-title}. With this update, the mTLS communication is enabled between Tempo components, the Operand starts successfully, and the Jaeger UI is accessible. (link:https://issues.redhat.com/browse/TRACING-3091[TRACING-3091])
+
+* Before this update, the resource limits from the {TempoOperator} caused errors such as *OOMKilled*. With this update, the resource limits for the {TempoOperator} are removed to avoid such errors. (link:https://issues.redhat.com/browse/TRACING-3204[TRACING-3204])
+
+=== Known issues
+
+* Currently, the custom TLS CA option is not implemented for connecting to object storage. (link:https://issues.redhat.com/browse/TRACING-3462[TRACING-3462])
+
+* Currently, when used with the {TempoOperator}, the Jaeger UI only displays services that have sent traces in the last 15 minutes. For services that did not send traces in the last 15 minutes, traces are still stored but not displayed in the Jaeger UI. (link:https://issues.redhat.com/browse/TRACING-3139[TRACING-3139])
+
+* Currently, the {TempoShortName} fails on the IBM Z (`s390x`) architecture. (link:https://issues.redhat.com/browse/TRACING-3545[TRACING-3545])
+
+* Currently, the Tempo query frontend service must not use internal mTLS when Gateway is not deployed. This issue does not affect the Jaeger Query API. The workaround is to disable mTLS. (link:https://issues.redhat.com/browse/TRACING-3510[TRACING-3510])
++
+.Workaround
+:!example-caption:
+====
+Disable mTLS as follows:
+
+. Open the {TempoOperator} ConfigMap for editing.
++
+[source,console]
+----
+$ oc edit configmap tempo-operator-manager-config -n openshift-tempo-operator <1>
+----
+<1> The project where the {TempoOperator} is installed.
+
+. Disable the mTLS in the Operator configuration.
++
+[source,yaml]
+----
+data:
+  controller_manager_config.yaml: |
+    featureGates:
+      httpEncryption: false
+      grpcEncryption: false
+      builtInCertManagement:
+        enabled: false
+----
+
+. Restart the {TempoOperator} Pod.
++
+[source,console]
+----
+$ oc rollout restart deployment.apps/tempo-operator-controller -n openshift-tempo-operator
+----
+====
+
+* Missing images for running the {TempoOperator} in restricted environments. The {TempoName} CSV is missing references to the operand images. (link:https://issues.redhat.com/browse/TRACING-3173[TRACING-3523])
++
+.Workaround
+:!example-caption:
+====
```

Comment:
This renders in a box because of `====`. Removing [136](https://github.com/openshift/openshift-docs/pull/62410/files#diff-873c33bdf9e924d854fea381b46c0255c61aa4c7a7692920f3080389d7cdd56fR136) and [162](https://github.com/openshift/openshift-docs/pull/62410/files#diff-873c33bdf9e924d854fea381b46c0255c61aa4c7a7692920f3080389d7cdd56fR162) should fix it

Reply:
Thank you. Applied manually.